plugins/tutor-contrib-learner-dashboard-mfe/tutor_learner_dashboard_mfe/plugin.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import annotations
+
+import os
+import os.path
+from glob import glob
+
+import pkg_resources
+
+from tutor import hooks
+from tutormfe.plugin import MFE_APPS
+
+########################################
+# CONFIGURATION
+########################################
+
+port = 1996
+
+@MFE_APPS.add()
+def _add_my_mfe(mfes):
+    mfes["learner-dashboard"] = {
+        "repository": "https://github.com/ghassanmas/frontend-app-learner-dashboard",
+        "port": port,
+        "version": "dynamic-config-tutor-mfe",
+    }
+    return mfes
+
+hooks.Filters.ENV_TEMPLATE_VARIABLES.add_items(
+    [("learner_dashboard_port", port)]
```

Comment:
Looking for thoughts here, @regisb: is there a better way to get the port to `openedx-lms-development-settings` with the new `MFE_APPS` mechanism?

Reply:
Sorry about the very late reply... If it were for me, I would just hardcode the port :-p But I understand it's not very elegant. Instead, we could add a `get_mfe` utility function to the `ENV_TEMPLATE_VARIABLES` in the tutor-mfe plugin:

```
def get_mfe(name: str):
    return MFE_APPS.get(name)

tutor_hooks.Filters.ENV_TEMPLATE_VARIABLES.add_item(("get_mfe", get_mfe))
```